docs/using/flowpipe/mods.md

```diff
@@ -70,6 +70,26 @@ applicable (if left empty, will default to `*`). When finished, click **Install
 
 After the mod is installed, you will be prompted to [set its variables](#managing-mod-variables).
 
+## Installing an Archived Mod
+
+You can install an archived mod by selecting **Install from mod archive** option. This will allow you to upload your own archived mod file subject to the following rules:
+
+- The mod archive must be one of the following formats: `.tar.gz`, `.tar`, `.zip`.
+- The archive must contain a `mod.pp` or `mod.sp` file at the root location and it should not be nested inside any folder.
+- Size of the archive should not exceed 20MB.
+
+Best practices to follow when creating an archived mod file:
+
+- Archive the mod file by navigating to the directory containing the `mod.pp` or `mod.sp` file to prevent the mod file from being nested inside any folder.
```

Comment:
I would think about using a code block and including the `cd {mod dir}` followed by the archive command...    Also, are these both `bash` commands?  do they work on mac and linux and windows??

Reply:
@johnsmyth I've made changes in the area to include a code block for the commands. 
For your other question, the commands work in mac and linux. However, for Windows, there's no in-built utility called `zip`, rather there's a `tar` utility which we can use to create the zip files.

Do we want to include instructions for Windows as well?

Reply:
Yes, I think we should.  I would use the code block switcher like we do in some other places, like [learn powerpipe ](https://powerpipe.io/docs) :
![image](https://github.com/user-attachments/assets/09dd5078-7ea3-4301-ba30-d0b123fbcd35)
https://github.com/turbot/powerpipe-docs/blob/main/docs/learn.md?plain=1#L28-L34